lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
```

Comment:
Agree that "telling Elixir" is maybe too informal.

The programmer should now that while public functions are callable, public macros are not unless you opt-in.

`require` could be renamed to `allow_me_to_call_public_macros_from`.

Let's see if we can come with a better wording!